packages/app/src/embed/components/Actions/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Tooltip from '@codesandbox/common/lib/components/Tooltip';
+import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
+
+import {
+  Container,
+  Button,
+  HeartIcon,
+  ReloadIcon,
+  NewWindowIcon,
+} from './elements';
+
+export function GlobalActions({
+  sandbox,
+  toggleLike,
+  previewVisible,
+  isDragging,
+}) {
+  return (
+    <Container
+      align="right"
+      previewVisible={previewVisible}
+      isDragging={isDragging}
+    >
+      {toggleLike ? (
+        <Tooltip
+          content={sandbox.userLiked ? 'Dislike sandbox' : 'Like sandbox'}
+        >
+          <Button
+            onClick={toggleLike}
+            aria-label={sandbox.userLiked ? 'Dislike sandbox' : 'Like sandbox'}
+          >
+            <HeartIcon liked={sandbox.userLiked} />
+          </Button>
+        </Tooltip>
+      ) : null}
+      <Button
+        as="a"
+        target="_blank"
+        rel="noopener noreferrer"
+        href={`${sandboxUrl(sandbox)}?from-embed`}
+      >
+        Open Sandbox
+      </Button>
+    </Container>
+  );
+}
+
+export function NavigationActions({ refresh, openInNewWindow, isDragging }) {
```

Comment:
Would be cool if we could move this to the Preview component itself in the future, we might be able to use this in the app in the future too!

Reply:
Good idea, would be cool to move this to common when the redesign is complete